integration/container/run_cgroupns_linux_test.go

```diff
@@ -38,7 +27,7 @@ func testRunWithCgroupNs(t *testing.T, daemonNsMode string, containerOpts ...fun
 	poll.WaitOn(t, container.IsInState(ctx, client, cID, "running"), poll.WithDelay(100*time.Millisecond))
 
 	daemonCgroup := d.CgroupNamespace(t)
-	containerCgroup := containerCgroupNamespace(ctx, t, client, cID)
+	containerCgroup := container.GetContainerNS(ctx, t, client, cID, "cgroup")
```

Comment:
This changes the code from reading `/proc/1/ns/cgroup` to `/proc/self/ns/cgroup` but the end result should be the same, so no issue here (just noticing).

Reply:
Yes, I know.